### PR TITLE
remove python2 support from awxkit

### PR DESCRIPTION
**Pull request by ryanpetrello**

### CODE REVIEW

File: awxkit/setup.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def run(self):
     },
     include_package_data=True,
     install_requires=requirements,
-    python_requires=">=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*",
+    python_requires=">=3.6",
```
Comment (ryanpetrello) [inline awxkit/setup.py:70]: 🎉

**Review by jbradberry — COMMENTED** [review]
Looks like you have to adjust the Makefile to only run tox with the Python 3 environment.

File: awxkit/setup.py (hunk 1 above)
Comment (jbradberry) [reply awxkit/setup.py:70]: Should we start adding Trove classifiers to the setup.py?  Are we intending to make awxkit, or at least awx-cli, available on pypi.org?

Comment (jbradberry) [reply awxkit/setup.py]: Should we start adding Trove classifiers to the setup.py?  Are we intending to make awxkit, or at least awx-cli, available on pypi.org?

Comment (ryanpetrello) [reply awxkit/setup.py:70]: We don't currently publish to PyPI (and don't have plans to in the _near_ future), but if we did in the future then we should definitely add Trove classifiers 👍

**Review by jakemcdermott — COMMENTED** [review]
https://github.com/ansible/awx/blob/devel/awxkit/awxkit/api/pages/credentials.py#L22

It'll continue to work fine, but this block can go if you feel like removing it.

File: awxkit/awxkit/cli/custom.py [hunk 2]
```diff
@@ -549,11 +547,8 @@ def perform(self, key, value):
         return resp.from_json({'key': key, 'value': resp[key]})
 
     def is_json(self, data):
-        err = ValueError
-        if PY3:
-            err = json.decoder.JSONDecodeError
         try:
             json.loads(data)
-        except err:
+        except json.decoder.JSONDecodeError:
```
Comment (chrismeyersfsu) [inline awxkit/awxkit/cli/custom.py:552]: :+1:

**Review by chrismeyersfsu — APPROVED** [review]

**Review by rooftopcellist — APPROVED** [review]

**Pull request merged**